test/unit/sequence_base_test.rb

```diff
@@ -48,4 +48,44 @@ def set_resource_reference(resource, type)
     new_reference.reference = "#{type}/1234"
     resource.recorder = new_reference
   end
+
+  describe '#retrieves value for search param' do
+    before do
+      instance = Inferno::Models::TestingInstance.create(selected_module: 'uscore_v3.0.0')
+      client = FHIR::Client.new('')
+      @sequence = Inferno::Sequence::SequenceBase.new(instance, client, true)
+    end
+
+    it 'returns value from period' do
+      element = FHIR::Period.new
```

Comment:
```suggestion
      element = FHIR::Period.new(start: 'now')
```
Would this sort of thing work for all these? If so, you could get a little fancy and really DRY things up by creating a hash of all the options you wanted to test and iterating over it.
```
{ state: mass, ... }.each do |key, value|
   element = FHIR::Address.new(key => value)
   ...
end
```

Reply:
Did it the DRY way, great suggestion.